kernels/ocn/do.bash

```diff
@@ -8,6 +8,9 @@ ln -s ../../wireguard-linux-compat/src net/wireguard
 [[ $(< net/Makefile) == *wireguard* ]] || sed -i "/^obj-\\\$(CONFIG_NETFILTER).*+=/a obj-\$(CONFIG_WIREGUARD) += wireguard/" net/Makefile
 [[ $(< net/Kconfig) == *wireguard* ]] ||  sed -i "/^if INET\$/a source \"net/wireguard/Kconfig\"" net/Kconfig
 
+# https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=ce513359d8507123e63f34b56e67ad558074be22
+patch -Np1 -i "$KERNEL_DIR/0001-scripts-dtc-Remove-redundant-YYLOC-global-declaratio.patch"
```

Comment:
It seems like `-N` will make it difficult to tell when the patch is no longer necessary and can be removed.

Reply:
Hmm, `patch -N` fails if the patch is already applied. I think that's enough to determine if the patch is no longer necessary?